docs/source/guide/guide-zne.rst

```diff
@@ -682,7 +682,13 @@ and clips the result if it falls outside its physical domain.
          self.min_expval = min_expval
          self.max_expval = max_expval
 
-      def reduce(self) -> float:
+      def extrapolate(
+         _,
```

Comment:
What's the reason for this?

Reply:
This can be avoided by adding the decorator `@staticmethod`. Not sure what the clearest/easiest way this should be handled.